examples/nemo_sdk/launch_slurm.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nemo_sdk as sdk
+from nemo_sdk.core.execution import SlurmExecutor
+
+from nemo_curator.nemo_sdk import SlurmJobConfig
+
+
+@sdk.factory
+def nemo_curator_slurm_executor() -> SlurmExecutor:
+    """
+    Configure the following function with the details of your SLURM cluster
+    """
+    return SlurmExecutor(
+        job_name_prefix="nemo-curator",
+        account="my-account",
+        nodes=2,
+        exclusive=True,
+        time="04:00:00",
+        container_image="nvcr.io/nvidia/nemo:dev",
+        container_mounts=["/path/on/machine:/path/in/container"],
+    )
+
+
+def main():
+    # Path to NeMo-Curator/examples/slurm/container_entrypoint.sh on the SLURM cluster
+    container_entrypoint = "/cluster/path/slurm/container_entrypoint.sh"
+    # The NeMo Curator command to run
+    curator_command = "text_cleaning --input-data-dir=/path/to/data --output-clean-dir=/path/to/output"
```

Comment:
Can this work for multiple commands ?

Reply:
Yes, this should be able to work for all dask-based curator scripts. It'll work for the rest too, though it'll be a bit overkill.

Reply:
Cool , we can leave it as is.  Maybe we add two.